addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/EnvFileScanRule.java

```diff
@@ -38,8 +49,31 @@ public int getId() {
 
     @Override
     public boolean isFalsePositive(HttpMessage msg) {
+        // We need the response header and the body
+        HttpResponseHeader responseHeader = msg.getResponseHeader();
+        String contentType = responseHeader.getNormalisedContentTypeValue();
         String responseBody = msg.getResponseBody().toString();
-        // It likely is a FP if the response contains neither a comment nor assignment
-        return !responseBody.contains("#") && !responseBody.contains("=");
+
+        // If we don't get a response header, the status code isn't 200 or content length is > 500,
```

Comment:
Or this one, its just repeating what the code says :)

Reply:
removed

Reply:
fixed now

Reply:
Maybe not pushed, the comment is still there.

Reply:
sorry thought you mean specific comments, pulled the other ones in there too